@here/harp-mapview/lib/Tile.ts

```diff
@@ -114,6 +115,26 @@ export interface TileFeatureData {
     objInfos?: Array<{} | undefined>;
 }
 
+/**
+ * Compute the memory footprint of `TileFeatureData`.
+ */
+export function getFeatureDataSize(featureData: TileFeatureData): number {
+    let numBytes = 100;
```

Comment:
Again, this initial estimation seems kind of weird. Please use constants in all similar places (will not add more comments for these).